src/components/PolicyTopologyPage.tsx

```diff
@@ -59,16 +373,51 @@ const PolicyTopologyPage: React.FC = () => {
           <Card>
             <CardTitle>Topology View</CardTitle>
             <CardBody>
+              <TextContent>
+                <Text component="p" className="pf-u-mb-md">
+                  This view visualizes the relationships and interactions between different
+                  resources within your cluster related to Kuadrant, allowing you to explore
+                  connections between Gateways, HTTPRoutes and Kuadrant Policies.
+                </Text>
+              </TextContent>
               {!loaded ? (
                 <div>Loading...</div>
               ) : loadError ? (
                 <div>Error loading topology: {loadError.message}</div>
               ) : parseError ? (
-                // parsing error
                 <div>Error parsing topology: {parseError}</div>
               ) : (
-                // render PolicyTopology if there are no errors in parsing
-                <PolicyTopology key={safeDotString} initialDotString={safeDotString} />
+                controller && (
+                  <TopologyView
+                    style={{ height: '70vh' }}
+                    className="kuadrant-policy-topology"
+                    controlBar={
+                      <TopologyControlBar
+                        controlButtons={createTopologyControlButtons({
+                          ...defaultControlButtonsOptions,
+                          zoomInCallback: action(() => {
+                            controller.getGraph().scaleBy(4 / 3);
+                          }),
+                          zoomOutCallback: action(() => {
+                            controller.getGraph().scaleBy(0.75);
+                          }),
+                          fitToScreenCallback: action(() => {
+                            controller.getGraph().fit(80);
+                          }),
+                          resetViewCallback: action(() => {
```

Comment:
Do we need a reset view if we have a fit to screen? They feel very similar when using. Reset view if you have the view in half screen looks kind of funky but works perfectly for fit to screen :
![Screenshot 2024-10-07 at 15 21 18](https://github.com/user-attachments/assets/0acec435-4a62-4c45-bd14-bb6d87c9f87c)

Reply:
had a similar thought earlier on, and this reminds me of it. I think fit is a better "reset" anyway

Reply:
removed, just fit as an action now